### PR TITLE
[en] Mention container orchestrator in k8s def

### PR DESCRIPTION
**Pull request by gdenn**

### CODE REVIEW

**Comment by gdenn** [comment]
issue: https://github.com/cncf/glossary/issues/707

File: content/en/kubernetes.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ category: technology
 ---
 
 ## What it is
-Kubernetes, often abbreviated as K8s, is a popular open-source tool for modern infrastructure automation. It's like a data center operating system that manages applications running across a [distributed system](/distributed_systems/) (just like the OS on your laptop that manages your apps). 
+Kubernetes, often abbreviated as K8s, is a container orchestrator which is an open-source tool that automates the life-cycle of [container](/container/) applications on modern infrastructures. It's like a data center operating system that manages applications running across a [distributed system](/distributed_systems/) (just like the OS on your laptop that manages your apps).
```
Comment (CathPag) [inline content/en/kubernetes.md:8]:
```suggestion
Kubernetes, often abbreviated as K8s, is an open source container orchestrator that automates the life-cycle of [containerized](/container/) applications on modern infrastructures. It's like a data center operating system that manages applications running across a [distributed system](/distributed_systems/) (just like the OS on your laptop that manages your apps). 
```

**Review by CathPag — COMMENTED** [review]
Good point, @gdenn! Two small edits.

File: content/en/kubernetes.md (hunk 1 above)
Comment (gdenn) [reply content/en/kubernetes.md:8]: thanks, @CathPag that sounds much better

**Review by CathPag — APPROVED** [review]
LGTM

**Review by iamNoah1 — APPROVED** [review]
lgtm, thanks @gdenn

**Pull request merged**